packages/pouchdb-ajax/src/request-browser.js

```diff
@@ -126,8 +126,7 @@ function xhRequest(options, callback) {
   try {
     xhr.open(options.method, options.url);
   } catch (exception) {
-   /* error code hardcoded to throw INVALID_URL */
-    callback(exception, {statusCode: 413});
+    return callback(new Error(exception.name || 'Url is invalid'));
```

Comment:
Yeh it probably should, ill fix that on the way in, also should clarify that I have to create a `new Error` because the exception given is readonly, which breaks a bunch of our code